assets/app/scripts/filters/resources.js

```diff
@@ -482,8 +482,10 @@ angular.module('openshiftConsole')
     };
   })
   .filter('projectOverviewURL', function(Navigate) {
-    return function(projectName) {
-      return Navigate.projectOverviewURL(projectName);
+    return function(project) {
+      return angular.isString(project) ?
+              Navigate.projectOverviewURL(project) :
+              Navigate.projectOverviewURL(project.metadata.name);
```

Comment:
True, I can update to:
```javascript
      return angular.isString(project) ?
              Navigate.projectOverviewURL(project) :
              Navigate.projectOverviewURL(project.metadata && project.metadata.name);
```
&& will check and ensure it doesn't generate something like `/project/undefined/overview`.